tests/test_simpleGenericMap.py

```diff
@@ -42,58 +42,84 @@ def makeMap(mapType, values):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.targets = {(str, SimpleGenericMapS)}
+        cls.implicitKeyTargets = {SimpleGenericMap}  # Must be mapping-constructible
+        cls.targets = {SimpleGenericMap[str]}  # Must be default-constructible
```

Comment:
Would you ever have more than one target in this `set`?

Reply:
In `targets`, certainly. `implicitKeyTargets` I'll admit is less likely. I guess I could make that one a scalar.